onnx/defs/gen_doc.py

```diff
@@ -9,13 +9,16 @@
 from collections import defaultdict
 
 from onnx import defs
-from onnx.defs import OpSchema
+from onnx.defs import OpSchema, FunctionProto
 from onnx.backend.test.case import collect_snippets
 from typing import Text, Sequence, Dict, List, Type, Set, Tuple
 
 
 SNIPPETS = collect_snippets()
 ONNX_ML = bool(os.getenv('ONNX_ML') == '1')
+ONNX_DOMAIN = ""
+ONNX_ML_DOMAIN = 'ai.onnx.ml'
```

Comment:
onnx.defs has `ONNX_DOMAIN` defined, maybe also move ONNX_ML_DOMAIN to there and here will just refer them through onnx.defs?